src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -783,6 +787,35 @@ private function getOrderSources(Order $order): OrderSourcesForViewing
         return new OrderSourcesForViewing($sources);
     }
 
+    /**
+     * @return LinkedOrdersForViewing
+     */
+    private function getLinkedOrders(Order $order): LinkedOrdersForViewing
+    {
+        $brothersData = $order->getBrother();
+        $brothers = [];
+        /** @var Order $brotherItem */
+        foreach ($brothersData as $brotherItem) {
+            $isTaxExcluded = !$this->isTaxIncludedInOrder($brotherItem);
+
+            if ($isTaxExcluded) {
+                $totalAmount = (string) $brotherItem->total_paid_tax_excl;
+            } else {
+                $totalAmount = (string) $brotherItem->total_paid_tax_incl;
+            }
+
+            $orderState = new OrderState($brotherItem->current_state);
+
+            $brothers[] = new LinkedOrderForViewing(
+                $brotherItem->id,
+                $orderState->name[$brotherItem->id_lang],
```

Comment:
I'm not sure we should use the Order language, If you're en English Employee managing a russian Order it might get confusing no?

Reply:
Very relevant feedback
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19704/commits/6cc29a04eae57adf48dc288aa3385fdab0ec3fff